config/routes.rb

```diff
@@ -26,6 +26,7 @@
     resources :specialties, only: [:new, :create, :index, :edit, :update]
     resources :tags, except: [:show]
     get '/dashboard', to: "admins#show"
+    patch '/rate_developer', to: 'developers#rate'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.